simulatorUser/use.rst

```diff
@@ -7,29 +7,30 @@ The main interface is a script to generate a single image.
 There is also a script that will generate a small dataset for testing the pipeline.
 
 The first time the Simulator is run, it will generate a PSF model and cache it;
-this can about an hour.
+this can take about an hour.
 Subsequent executions will be much faster.
 
+Initial setup
+-------------
 
-Single image
-------------
+``makePfsDesign``
+~~~~~~~~~~~~~~~~~
 
-The ``makeSim`` script runs the Simulator to produce a single image
-and the associated ``pfsConfig`` file.
-An example command-line is::
+The simulator requires a ``pfsDesign`` (or ``pfsConfig``) 
+file to tell it for which fibers are assigned 
+to which sources. If there is not such a file available, one can generate that using the
+``makePfsDesign`` command::
 
-    makeSim --detector r1 --field lamObj --exptime 900 --expId 32 --imagetyp OBJECT --allOutput --pdb
+    makePfsDesign --fibers lam --pfsDesignId 1 --scienceCatId 1 --scienceObjId "18 55 71 76 93 94 105 112 115"
 
-The ``--detector`` specifies the spectrograph and arm;
-we currently only support ``r1``.
+The above command generates a file ``pfsDesign-0x0000000000000001.fits``, corresponding to a
+``pfsDesignId`` of ``1``, 
+mapping randomly objects with object identifiers listed in the value for the 
+``--scienceObjId`` option. These object identifiers are associated with a catalogue '``1``' 
+(the value of ``--scienceCatId``). 
```

Comment:
The simulator will then look under the directory `1` (the `scienceCatId`) for the input spectra.

Reply:
Fixed.